domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterSingleton.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.android.domain.platformparameter
+
+import javax.inject.Inject
```

Comment:
Re: linter errors, I believe the `javax` imports should go at the end.

Reply:
Done